Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -318,9 +330,9 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
 
     Ref<WebExtensionStorageSQLiteStore> storageForType(WebExtensionDataType);
 
-    bool load(WebExtensionController&, String storageDirectory, NSError ** = nullptr);
-    bool unload(NSError ** = nullptr);
-    bool reload(NSError ** = nullptr);
+    bool load(WebExtensionController&, String storageDirectory, RefPtr<API::Error>&);
+    bool unload(RefPtr<API::Error>&);
+    bool reload(RefPtr<API::Error>&);
```

Comment:
This is where `Expected<bool, RefPtr<API::Error>>` results would help, so error can be ignored if needed now that it is a reference.